### PR TITLE
Add `version` command

### LINKED ISSUE #148: [Feature Request] Add `version` console command to display the server version
**Is your feature request related to a problem? Please describe.**
It would make it easier to maintain and update the server if I could easily check the version with a command. Software like Pterodactyl may not allow you to scroll up far in the console

**Describe the solution you'd like**
a `version` command in the server console

**Describe alternatives you've considered**
Displaying the version number in the `status` command

### PR DESCRIPTION
**Pull request by sla-ppy**
closes #148

### CODE REVIEW

File: src/TConsole.cpp [hunk 1]
```diff
@@ -247,7 +247,8 @@ void TConsole::Command_Help(const std::string&, const std::vector<std::string>&
         lua [state id]          switches to lua, optionally into a specific state id's lua
         settings [command]      sets or gets settings for the server, run `settings help` for more info
         status                  how the server is doing and what it's up to
-        clear                   clears the console window)";
+        clear                   clears the console window
+        version                 return current version to console)";
```
Comment (lionkor) [inline src/TConsole.cpp]: Would be better if the wording here more closely matched the other help strings, like "displays this help", perhaps like:
```
        version                 displays the version of this server)";
```

File: src/TConsole.cpp [hunk 2]
```diff
@@ -267,6 +268,14 @@ void TConsole::Command_Clear(const std::string&, const std::vector<std::string>&
     mCommandline->write("\x1b[;H\x1b[2J");
 }
 
+void TConsole::Command_Version(const std::string& cmd, const std::vector<std::string>& args) {
+    if (!EnsureArgsCount(args, 0)) {
+        return;
+    }
+
+    mCommandline->write("Current version: " + Application::ServerVersionString());
```
Comment (lionkor) [inline src/TConsole.cpp]: For consistency, we could use `Application::Console().WriteRaw()` or just `WriteRaw()` instead... Not sure why the other functions are inconsistent about the use of this as well.

Maybe just changing this to use `Application::Console().WriteRaw()` might be nice, to at least be consitent with most of the other commands. A separate issue can be opened to address this inconsistency and the weird way in which it's called (via Application::?? why???? who knows.)

**Review by lionkor — CHANGES_REQUESTED** [review]

**Comment by lionkor** [comment]
Thanks for your contribution! Please see if you can make the requested changes, and then request a review from me again :)

**Comment by sla-ppy** [comment]
@lionkor Please review my changes so the issue can be finalized.

File: src/TConsole.cpp [hunk 3]
```diff
@@ -267,6 +268,14 @@ void TConsole::Command_Clear(const std::string&, const std::vector<std::string>&
     mCommandline->write("\x1b[;H\x1b[2J");
 }
 
+void TConsole::Command_Version(const std::string& cmd, const std::vector<std::string>& args) {
+    if (!EnsureArgsCount(args, 0)) {
+        return;
+    }
+
+    Application::Console().WriteRaw("Current version: " + Application::ServerVersionString());
```
Comment (lionkor) [inline src/TConsole.cpp]: could we get a:
```suggestion
    Application::Console().WriteRaw("Current version: v" + Application::ServerVersionString());
```
that way its consistent with the other version printouts

**Review by lionkor — CHANGES_REQUESTED** [review]
pls change this as well :)

**Review by lionkor — APPROVED** [review]
lgtm

**Pull request merged**